samples/metaapp/src/main/res/layout-television/property_list_title.xml

```diff
@@ -55,6 +42,16 @@
         app:layout_constraintStart_toEndOf="@+id/removeAllBtn"
         app:layout_constraintTop_toTopOf="parent" />
 
+    <TextView
+        android:id="@+id/titleTextView"
+        android:layout_width="wrap_content"
+        android:layout_height="64dp"
+        android:layout_centerVertical="true"
+        android:focusable="false"
+        android:gravity="center"
+        android:textAppearance="@android:style/TextAppearance.Large"
```

Comment:
use the following style for the title 
https://github.com/SourcePointUSA/android-cmp-app/blob/1c034b128d7c176e42f1684bac61a3bf62787374/samples/metaapp/src/main/res/layout-television/demo_header.xml#L24